### PR TITLE
Add image metadata, test and release notes

### PR DESCRIPTION
**Pull request by mirr254**
### What does this PR do?
Tests the current image and adds the required metadata and release notes.

### Description of work done
- Update the image manifests with `LABEL` values
- Test the image against mockbin.org API testing suite

### How should this be manually tested
- The easiest way I found was to use docker-compose to have all the services. i.e `db(postgres)`, `db-migration` and `gg`
- `gg` has
```
    depends_on:
      gg-database:
        condition: service_healthy
      gg-migration:
        condition: service_started
```
- `db-migration` has
```
    depends_on:
      gg-database:
        condition: service_healthy
```
- run `docker-compose exec gg /bin/sh` to get a shell.

### Screenshots (If available)
Adding a service using admin API
Screenshot 2019-11-19 at 13 03 49

Adding a route for that service

Screenshot 2019-11-19 at 13 06 10

Confirming if we can forward requests through our image

Screenshot 2019-11-19 at 13 09 18

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ RUN apk update \
 # Gluu Gateway
 # ============
 
-ENV GLUU_VERSION=version_4.0 \
+ENV GLUU_VERSION=version_4.0.0 \
```
Comment (altexy) [inline Dockerfile]: Did you try to build with this change?

from rocketchat:

> Version 4.0 is fixed with v4.0.0 tag. You may create a branch and publich the 4.0.0 image.
> Just use GLUU_VERSION=v4.0.0 in Dockerfile

Comment (mirr254) [reply Dockerfile]: I may have misunderstood your comment Alex. Let me rectify that.

**Comment by mirr254** [comment]
Closing in favor of (pull #4 )

**Comment by mirr254** [comment]
Closing in favor of pull #4

**Pull request closed without merging**